core/web/schema/type/feeds_manager.graphql

```diff
@@ -77,14 +77,22 @@ type CreateFeedsManagerSuccess {
     feedsManager: FeedsManager!
 }
 
+type DuplicateFeedsManagerError implements Error {
+	message: String!
+	code: ErrorCode!
+}
+
+# DEPRECATED: No longer used since we now support multiple feeds manager.
+# Keeping this to avoid breaking change.
 type SingleFeedsManagerError implements Error {
 	message: String!
 	code: ErrorCode!
 }
 
 # CreateFeedsManagerPayload defines the response when creating a feeds manager
 union CreateFeedsManagerPayload = CreateFeedsManagerSuccess
-	| SingleFeedsManagerError
+	| DuplicateFeedsManagerError
+	| SingleFeedsManagerError # // TODO: delete once multiple feeds managers support is released
```

Comment:
Is there a ticket to link for these?

Reply:
Do you mean a ticket to cleanup the `SingleFeedsManagerError` ?